src/features/metadata-instance-editor/flowTypes.js

```diff
@@ -0,0 +1,107 @@
+// @flow
```

Comment:
I'm surprised this was previously omitted. Yikes.

Reply:
Remove this file, the types should be available globally